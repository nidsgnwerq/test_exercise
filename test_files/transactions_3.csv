type,client,tx,amount
deposit,1,10001,2000.000
deposit,1,10002,1000.000
dispute,1,10002,
chargeback,1,10002,

