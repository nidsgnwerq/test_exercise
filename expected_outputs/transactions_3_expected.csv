client, available, held, total, locked
1, 2000.0, 0.0, 2000.0, True
